### PR TITLE
Fix cli tests after the addition of acknowledgement of disclaimer

### PR DESCRIPTION
**Pull request by LefterisJP**
[ci integration]

### CODE REVIEW

File: raiden/tests/integration/cli/test_cli.py [hunk 1]
```diff
@@ -114,13 +121,11 @@ def test_cli_wrong_rpc_endpoint(cli_args):
...
     try:
-        child.expect('Welcome to Raiden')
-        child.expect('The following accounts were found in your machine:')
-        child.expect('Select one of them by index to continue: ')
-        child.sendline('0')
+        expect_cli_until_account_selection(child)
         child.expect(
             "The chosen ethereum network 'mainnet' differs from the ethereum "
-            "client 'smoketest'")
+            "client 'smoketest'"
```
Comment (andrevmatos) [inline raiden/tests/integration/cli/test_cli.py]: Lint complaining about the missing comma here

Comment (LefterisJP) [reply raiden/tests/integration/cli/test_cli.py]: ofcourse it does -.-

Comment (LefterisJP) [reply raiden/tests/integration/cli/test_cli.py]: I really don't get it why locally I am not getting these errors when I run flake8.

**Comment by LefterisJP** [comment]
merged because it does fix the CLI tests

**Pull request merged**